pinot-controller/src/test/java/com/linkedin/pinot/controller/helix/ControllerTest.java

```diff
@@ -137,6 +144,30 @@ protected void stopController() {
     _zkClient.close();
   }
 
+  protected void addDefaultSchema(String tableName) throws JSONException, IOException {
+    JSONObject schemaJson = new JSONObject();
```

Comment:
You can build the schema object and call `schema.getJSONSchema()` to get json string.